doc/dune-files.rst

```diff
@@ -801,17 +801,36 @@ Fields supported in ``<settings>`` are:
 ignored_subdirs
 ---------------
 
-The ``ignored_subdirs`` stanza allows to tell Dune to ignore one or
-more sub-directories. The syntax is as follow:
+The ``ignored_subdirs`` stanza allows to tell Dune to ignore one or more
+sub-directories. The syntax is based on dune's predicate language and allows the
+user the following operations:
+
+- The special value ``:standard`` which refers to the default set of ignored
+  directories. These are the directories that start with any of ``[._#]``.
```

Comment:
```suggestion
  directories. These are the directories that start with ``.`` or ``_``.
```

Otherwise it might not be clear that this is a glob. BTW, why `#`? We didn't use to ignore directories starting with `#`.